cypress-smoketests/support/pages/login.js

```diff
@@ -51,27 +53,28 @@ const getUserToken = async (username, password) => {
     .promise();
 };
 
-let token = null;
+exports.login = token => {
+  cy.visit(`/log-in?token=${token}`);
 
-describe('petitionsclerk can login', () => {
-  before(async () => {
-    const results = await getUserToken(
-      'petitionsclerk1@example.com',
-      'Testing1234$',
-    );
-    token = results.AuthenticationResult.IdToken;
-  });
+  cy.get('.progress-indicator').should('not.exist');
+};
 
-  it('views the section inbox', () => {
-    cy.visit(`/log-in?token=${token}`);
-    cy.get('.button-switch-box').should('exist');
+exports.getRestApi = async () => {
+  let apigateway = new AWS.APIGateway({
+    region: awsRegion,
   });
+  const { items: apis } = await apigateway
+    .getRestApis({ limit: 200 })
+    .promise();
 
-  it('should verify advanced search works', () => {
-    cy.visit(`/log-in?token=${token}&path=/search`);
-    cy.get('#advanced-search-button').should('exist');
-    cy.get('#petitioner-name').type('THISNAMESHOULDNEVEREXIST');
-    cy.get('#advanced-search-button').click();
-    cy.get('#no-search-results').should('exist');
-  });
-});
+  const services = apis
+    .filter(api => api.name.includes(`gateway_api_${ENV}`))
+    .reduce((obj, api) => {
+      obj[
+        api.name.replace(`_${ENV}`, '')
+      ] = `https://${api.id}.execute-api.${awsRegion}.amazonaws.com/${ENV}`;
```

Comment:
@ericsorenson I feel like there's potential here to make the whole url configurable.

Reply:
@julialeague can you be more specific about what you're looking for?

Reply:
@ericsorenson I was thinking that the url as a whole could be stored in a config file and updated there as needed - what if the AWS url pattern changes or something?